velox/docs/develop/types.rst

```diff
@@ -58,6 +58,17 @@ UNKNOWN type is used to represent an empty or all nulls vector of unknown type.
 For example, SELECT array() returns an ARRAY(UNKNOWN()) because it is not possible
 to determine the type of the elements. This works because there are no elements.
 
+TIMESTAMP type is used to represent a specific point in time.
+A TIMESTAMP is defined as the sum of seconds and nanoseconds since UNIX epoch.
+`struct Timestamp` uses a 64-bit integers to store seconds and a 64-bit unsigned integer
+to store nanoseconds. Nanoseconds in Timestamp is used to represent the high-precision
+part of Timestamp less than 1 second, so the value range of nanoseconds is [0, 10^9).
+For example, `Timestamp(0, 0)` represents `1970-01-01 00:00:00` and
```

Comment:
Let's put examples into a separate paragraph and format them one per line for readability:

```
Examples:

* Timestamp(0, 0) represents 1970-01-01 00:00:00 (epoch).
* Timestamp(10*24*60*60 + 125, 0) represents 1970-01-11 00:02:05 (10 days 125 nanoseconds after epoch).
* Timestamp(-(10*24*60*60 + 125), 0) represents 1969-12-21 23:57:55 (10 days minus 125 nanoseconds before the epoch).
```